presentation/src/main/java/com/yapp/bol/presentation/view/match/member_select/MemberSelectFragment.kt

```diff
@@ -85,6 +97,22 @@ class MemberSelectFragment : Fragment() {
         matchViewModel.players.observe(viewLifecycleOwner) { players ->
             memberSelectAdapter.submitList(players)
         }
+
+        matchViewModel.isNickNameValidate.observe(viewLifecycleOwner) {
+            if (dialog.isShowing) dialog.setNicknameValid(it)
+        }
+    }
+
+    private fun setSearchResultNothing(isVisible: Boolean, keyword: String) {
+        val visible = if (isVisible) View.VISIBLE else View.GONE
+        val searchResult = String.format(resources.getString(R.string.search_result_nothing), keyword)
+        binding.viewSearchResultNothing.visibility = visible
+        binding.tvSearchResultNothingGuide.visibility = visible
+        binding.btnGuestAddNothing.visibility = visible
+        binding.tvSearchResultNothing.apply {
+            text = searchResult
+            visibility = visible
+        }
     }
```

Comment:
```suggestion
    private fun setSearchResultNothing(isVisible: Boolean, keyword: String) {
        val searchResult = String.format(resources.getString(R.string.search_result_nothing), keyword)
        binding.viewSearchResultNothing.isVisible = isVisible
        binding.tvSearchResultNothingGuide.isVisible = isVisible
        binding.btnGuestAddNothing.isVisible = isVisible
        binding.tvSearchResultNothing.apply {
            text = searchResult
            this.isVisible = isVisible
        }
    }
```
`isVisible` 을 사용하면 따로 프로퍼티를 안만들어도 될 것 같아요!